ocean_lib/data_provider/data_service_provider.py

```diff
@@ -282,23 +292,27 @@ def start_compute_job(
         payload = DataServiceProvider._prepare_compute_payload(
             consumer=consumer,
             dataset=dataset,
-            compute_environment=compute_environment,
+            compute_environment=json.dumps(
```

Comment:
are you sure? As far as I know this compute environment is decided by the user by querying the provider options, not from the dataset itself. Please reinstate this.

Reply:
This is still an issue.